tests/entry.rs

```diff
@@ -62,12 +62,22 @@ fn absolute_hardlink() {
     header.set_cksum();
     t!(ar.append(&header, &[][..]));
 
+    let mut header = tar::Header::new_gnu();
+    header.set_size(0);
+    header.set_entry_type(tar::EntryType::Link);
+    t!(header.set_path("baz"));
+    // This absolute path on root will be converted when unpacking
+    t!(header.set_link_name("/foo"));
+    header.set_cksum();
+    t!(ar.append(&header, &[][..]));
```

Comment:
There's a couple other branches through the `if` statement added above, could tests be added for the other branches as well?

Reply:
Actually the existing test (`foo`) is already testing the other case:  absolute path starting with the target directory.